contrib/ansible/group_vars/all.yml

```diff
@@ -6,7 +6,7 @@ cluster_name: cluster.local
 # Account name of remote user. Ansible will use this user account to ssh into
 # the managed machines. The user must be able to use sudo without asking
 # for password unless ansible_sudo_pass is set
-#ansible_ssh_user: root
+ansible_ssh_user: fedora
```

Comment:
as long as all.yml exists, in the current form, it will render any automation not using those exact defaults impossible.    are we sure we want to mandate that?